integration-tests/src/test/java/oracle/weblogic/kubernetes/ItRetryImprovementMisc.java

```diff
@@ -0,0 +1,471 @@
+   * Max cluster size set in the model file is 5.
+   *
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   *           JUnit engine parameter resolution mechanism
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(2) List<String> namespaces) {
+    logger = getLogger();
+
+    // get a namespace for operator
+    assertNotNull(namespaces.get(0), "Namespace namespaces.get(0) is null");
+    opNamespace = namespaces.get(0);
+
+    // get a namespace for domain1
+    assertNotNull(namespaces.get(1), "Namespace namespaces.get(1) is null");
+    domainNamespace = namespaces.get(1);
+
+    opServiceAccount = opNamespace + "-sa";
+    // install and verify operator with REST API
+    logger.info("Install an operator in namespace {0}, managing namespace {1}",
+        opNamespace, domainNamespace);
+    installAndVerifyOperator(opNamespace, opServiceAccount, true, 0, domainNamespace);
+  }
+
+  /**
+   * Start a domain with a single cluster, where cluster has 1 server (replicas = 1).
+   * Expect Cluster and Domain status to report Unavailable and Incomplete almost immediately.
+   * Wait for Cluster and Domain status to both report Available=True and Complete=True,
+   * and expect this to happen at the same time all pods reports ready (not sooner).
+   */
+  @Order(1)
+  @Test
+  @DisplayName("Start a domain with a single cluster which has 1 server")
+  void testStartSingleClusterDomainWithOneServer() {
+    // create a mii domain resource with one cluster
+    logger.info("Create model-in-image domain {0} in namespace {1}, and wait until it comes up",
+        domainUid, domainNamespace);
+    createMiiDomain(
+        domainNamespace,
+        domainUid,
+        MII_BASIC_IMAGE_NAME + ":" + MII_BASIC_IMAGE_TAG,
+        replicaCount,
+        Arrays.asList(clusterName),
+        false,
+        null);
+
+    // check the domain status is Unavailable and Incomplete
+    checkDomainStatusConditionTypeHasExpectedStatus(domainUid, domainNamespace,
+        DOMAIN_STATUS_CONDITION_COMPLETED_TYPE, "False");
+    checkDomainStatusConditionTypeHasExpectedStatus(domainUid, domainNamespace,
+        DOMAIN_STATUS_CONDITION_AVAILABLE_TYPE, "False");
+
+    // check the cluster status is Unavailable and Incomplete
+    checkDomainStatusClusterConditionTypeHasExpectedStatus(domainUid, domainNamespace, clusterName,
+        DOMAIN_STATUS_CONDITION_COMPLETED_TYPE, "False");
+    checkDomainStatusClusterConditionTypeHasExpectedStatus(domainUid, domainNamespace, clusterName,
+        DOMAIN_STATUS_CONDITION_AVAILABLE_TYPE, "False");
+
+    // wait for all the pods are up
+    // check admin server pod is ready
+    logger.info("Wait for admin server pod {0} to be ready in namespace {1}",
+        adminServerPodName, domainNamespace);
+    checkPodReadyAndServiceExists(adminServerPodName, domainUid, domainNamespace);
+
+    // check the domain status is Unavailable and Incomplete
```

Comment:
Even after the service and pod is available, why the completed status condition is false ? Is it due to reason the cluster service is not up ? If so please add the proper description